workloads/burn-in/install_hplbit_deps.sh

```diff
@@ -0,0 +1,239 @@
+   *) echo "Unable to find a matching MOFED version for HPCX, exiting." & exit 1 ;;
+esac
+
+#sets the directory where the burn in test is located, then sets where CUDA and HPC-X will downloaded and built
+export BASEDIR=${BASEDIR:-$(cd $(dirname 0) && pwd)}
+export BUILDDIR=${BUILDDIR:-/tmp/build.$$}
+#echo $BASEDIR
+#echo $BUILDDIR
+
+APPSDIR=${BASEDIR}/apps
+
+mkdir -p $APPSDIR
+mkdir -p $BUILDDIR
+
+
+#Variable definitions for CUDA and HPC-X Installs
+
+#CUDA Install Variables
+CUDA_VERSION=11.0.2
+CUDA_HOME=${APPSDIR}/cuda/${CUDA_VERSION}
+mkdir -p ${CUDA_HOME}
+#echo $CUDA_HOME
+
+#HPCX Install Variables
+
+#HPCX_URL="http://www.mellanox.com/downloads/hpc/hpc-x/v2.7/hpcx-v2.7.0-gcc-MLNX_OFED_LINUX-5.0-1.0.0.0-ubuntu18.04-x86_64.tbz"
+HPCX_VERSION=$(echo $HPCX_URL | grep -o '[^/]*$' | cut -f5-6 -d-)
+#echo $HPCX_VERSION
+HPCX_FN=$(basename ${HPCX_URL})
+HPCX_DIR=$(basename ${HPCX_FN} .tbz)
+
+#Build CUDA Toolkit and HPC-X if selected to do so (Yes by default)
+
+#Build CUDA Toolkit from Run File
+if [ $BUILD_CUDA == 1 ]; then
+    echo ""
+    echo "INFO: Installing CUDA Toolkit ${CUDA_VERSION}"
+    echo ""
+
+    CLOG=/tmp/cuda-installer.log
+    if [ -f ${CLOG} ]; then
+	    echo ""
+	    echo "INFO: The file ${CLOG} was found, trying to remove".
+	    rm -f ${CLOG}
+            if [ -f ${CLOG} ]; then
+		    echo ""
+		    echo "ERROR: Unable to remove ${CLOG} before installation of CUDA."
+		    echo "ERROR: Existence of this file without write permission will"
+
+echo "ERROR: cause the instalation of CUDA to trigger a segementation"
+		    echo "ERROR: fault.  Please have the file removed, and try the"
+		    echo "ERROR: Installation again."
+		    echo ""
+		    exit 1
+            fi
+	    echo "INFO: File ${CLOG} has been removed."
+	    echo ""
+    fi
+
+#Change to build directory where the CUDA run file is downloaded.  Remove the file if it is already present. Then run the installer to install the toolkit only. 
+    cd ${BUILDDIR}
+    CUDAPATH=http://developer.download.nvidia.com/compute/cuda/11.0.2/local_installers/cuda_11.0.2_450.51.05_linux.run
+    CUDAFN=$(basename ${CUDAPATH})
+    rm -f ${CUDAFN}
+    wget http://developer.download.nvidia.com/compute/cuda/11.0.2/local_installers/${CUDAFN}
+
+    sh ${CUDAFN} --installpath=${CUDA_HOME} --silent --toolkit
+
+    if [ $? -ne 0 ]; then
+	echo "ERROR: Install of CUDA failed.  Exiting"
+   fi
+   echo ""
+   echo "INFO: Done installing CUDA"
+fi
+
+
+#Build HPC-X and install UCX and OpenMPI
+if [ $BUILD_HPCX == 1 ]; then
+	# First some sanity checking to make sure that the right version
+	echo ""
+	echo "INFO: Installing HPCX for MLNX OFED ${HPCX_VERSION}"
+
+#On Login nodes where there is no mellanox adapter, this check will fail.  Perhaps do an srun to use a compute node on the test cluster and check its OFED, since it will be the same as all the nodes in the POD
```

Comment:
Can we delete all this commented out stuff or is it useful?

Reply:
Yes

Reply:
The commented out stuff here is mostly check we put in that could be useful in a future revision depending on how the test plays out in the field.  However if it is preferred that such code is removed, i can work to do so before merging the PR